pkg/parser/ast.go

```diff
@@ -394,6 +400,18 @@ func (d *DotExpression) Type() *Type {
 	return d.T
 }
 
+func (t *TypeAssertion) Token() *lexer.Token {
+	return t.token
+}
+
+func (t *TypeAssertion) String() string {
+	return "(" + t.Left.String() + "." + "(" + t.T.String() + ")" + ")"
```

Comment:
Hmmm, should I be using `Sprintf` here now?

Reply:
Up to you, I think this is fine.